lib/fog/google/requests/storage_json/put_object_acl.rb

```diff
@@ -0,0 +1,49 @@
+module Fog
+  module Google
+    class StorageJSON
+      class Real
+        # TODO: move this methods to helper to use them with put_bucket_acl request
+        def tag(name, value)
+          # "<#{name}>#{value}</#{name}>"
+        end
+
+        def scope_tag(scope)
```

Comment:
Unused method argument - `scope`. If it's necessary, use `_` or `_scope` as an argument name to indicate that it won't be used. You can also write as `scope_tag(*)` if you want the method to accept any arguments but don't care about them.